apis/kueue/v1beta1/workload_types.go

```diff
@@ -327,6 +327,10 @@ const (
 
 	// WorkloadRequeued means that the Workload was requeued due to eviction.
 	WorkloadRequeued = "Requeued"
+
+	// WorkloadWaitingForReplacementPods is True when Kueue doesn't observe all
+	// the Pods declared for the group.
+	WorkloadWaitingForReplacementPods = "WaitingForReplacementPods"
```

Comment:
let's put this in the pod controller for now, given that it's not universal to other integrations, and the pod integration is disabled by default.

Reply:
Moved